### DISCUSSION TITLE
Is it safe?

### DISCUSSION THREAD

**Discussion by beshur** [danger/danger-js#1298] [Q&A] [answered]
Hi.

Can I trust this library? 
It has quite a large codebase, I cannot reliably verify that it does not sniff on my code in CI and send it somewhere else. 
Also any kind of modifications could have been done to it on package generation step.

// Yeah, I know it's kind of a strange question for a library named "danger"

**Comment by orta** [comment, marked as answer]
It is a strange question, yep. 

You can trust it as far as you can trust any dependency in your codebase. Audit the code and make your own decision Anything I say wouldn't be useful in changing that opinion.

**Comment by beshur** [comment]
Right, thank you for reply.
